resources/translations/fr_FR.ts

```diff
@@ -0,0 +1,302 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="en_AU" sourcelanguage="en_AU">
+<context>
+    <name>AboutPage</name>
+    <message>
+        <location filename="../qml/pages/AboutPage.qml" line="14"/>
+        <source>Shmoose - Xmpp Client</source>
+        <translation>Shmoose - Client Xmpp</translation>
+    </message>
+    <message>
+        <location filename="../qml/pages/AboutPage.qml" line="27"/>
+        <source>Version</source>
+        <translation>Version</translation>
+    </message>
+    <message>
+        <location filename="../qml/pages/AboutPage.qml" line="34"/>
+        <source>Contributions:</source>
+        <translation>Contributions :</translation>
```

Comment:
Is there really a space between a word and the colon in french?

Reply:
is not a space but a espace fine insécable : https://fr.wikipedia.org/wiki/Espace_fine_ins%C3%A9cable
It's half a space and the colon cannot be put on another line.

Reply:
I put a space because that's the character everyone uses when typing and weird unicode space characters have a tendency to break things and not be seen